tidb/src/main/java/io/tidb/bigdata/tidb/JdbcConnectionProviderFactory.java

```diff
@@ -28,6 +28,7 @@ public class JdbcConnectionProviderFactory {
 
   public static JdbcConnectionProvider createJdbcConnectionProvider(ClientConfig config) {
     try {
+      Class.forName(ClientConfig.MYSQL_DRIVER_NAME);
```

Comment:
What's the purpose

Reply:
It is a static function in com.mysql.jdbc.driver that register the MySQL driver to JAVA driver manager.
![7f2pLCJXJl](https://user-images.githubusercontent.com/35677990/173525588-5624286e-7396-4cf7-8bb3-388225f9fe79.png)
Here is a link about [register driver](https://stackoverflow.com/questions/9575367/about-jdbc-drivers).

Reply:
You can click to find an [official description of DriverManager](https://docs.oracle.com/javase/8/docs/api/java/sql/DriverManager.html).

Reply:
Add a comment, load class will cause the driver to register itself to DriverMananger

Reply:
OK, I have added a comment to explain the `Class.forName(JDBC_Driver)`.

Reply:
if we don't register JDBC driver, it will register h2 driver in default?

Reply:
@shiyuhang0 Now, DriverManager would find the JDBC class name in `META-INF/services/java.sql.Driver` file. It will register h2db auto.

Reply:
Please add the method link, otherwise, it's still difficult to understand how to register

Reply:
yeah, That is confusing. I add a link to `com.mysql.cj.jdbc.Driver`.